tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  position: relative;
+  height: 100vh;
+  width: 100%;
```

Comment:
Give this `viewport width` as you had given to height

Reply:
Done @shaikh-7abish